docs/architecture/workflow-update.md

```diff
@@ -131,15 +131,6 @@ via `newAccepted()`. Similarly, when an Update is resurrected (see "Update Resur
 also has state `Admitted`. But when an Update cannot be found in the Registry, yet it exists in
 `EventStore` because it is already completed, it is created with state `Completed`.
 
```

Comment:
Question on code above:

> Since rejected Updates are not stored anywhere and, therefore, cannot be deduplicated they can be shipped to the worker twice.

When do we ship rejected updates to the worker?

Reply:
Not "rejected update", but "update that previously was rejected". Imagine, worker rejects, server doesn't send it to client, client retries, server sends same update to worker again.